### PR TITLE
Database seed completed

### PR DESCRIPTION
**Pull request by kcords**
Database seed ready for review.

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
-connect('mongodb://localhost/', { useNewUrlParser: true, useUnifiedTopology: true })
+mongoose.connect('mongodb://localhost/fec', { useNewUrlParser: true, useUnifiedTopology: true })
+  // eslint-disable-next-line no-console
   .then(() => console.log('Successfully connected to MongoDB!'))
-  .catch(err => console.log('Error connecting to the database.', err));
+  // eslint-disable-next-line no-console
+  .catch((err) => console.log('Error connecting to the database.', err));
 
-const photoSchema = new Schema({
+mongoose.Promise = global.Promise;
```
Comment (juanoxmar) [inline database/index.js]: current mongoose natively returns promises. Promsifying it is not needed.

**Review by juanoxmar — CHANGES_REQUESTED** [review]
Just the global promise for mongoose

**Review by juanoxmar — APPROVED** [review]
Looks good!

File: database/index.js (hunk 1 above)
Comment (kcords) [reply database/index.js]: Thank you for bringing this to my attention. I meant to remove this prior to the previous commit. This has now been removed and resubmitted for review.

**Review by juanoxmar — APPROVED** [review]
Cool!

**Comment by kcords** [comment]
[Create a seed script for database](https://trello.com/c/Tq73ctfg/5-create-a-seed-script-for-database)

**Pull request merged**